### DISCUSSION TITLE
Is updating a dropdown menu based on another menu's selection still not possible? (March 2024)

### DISCUSSION THREAD

**Discussion by bertiebaggio** [vega/altair#3375] [Q&A] [answered]
**Problem**: a visualisation designer wants to let the user make some selection / filter on data at an arbitrary level.

**Examples**: 

Stock prices: letting the user select a type/sector ('tech, manufacturing, ...,' ), then a stock ('GOOG, MSFT, ...,' / 'BBY, WEIR, ...')

Dataset with  geographical indicators: let the user select a geographical level (continent, country, province, municipality) then presenting the choices at that level (eg 'Europe, Asia, Africa, ...' / 'Bulgaria, Greece, North Macedonia, ...' → 'Sofia City Province, Sofia Province, Plovdiv Province, Montana Province, ...' etc).

[A 2020 SO answer by javkevdp](https://stackoverflow.com/a/59988451/4647265) suggests this was not possible then. There are no relevant discussions here for 'dropdown', and none at all for 'menu'.

**Question**: is it possible to update one dropdown menu based on the selection of another for the purposes of filtering data?

**Comment by joelostblom** [comment, marked as answer]
Hello @bertiebaggio, This is not possible in Vega-Lite/Altair. As Jake wrote at the end of the answer you linked; this type of feature would need to be added in Vega-Lite before it becomes available in Altair, so please search through [the issues there](https://github.com/vega/vega-lite/issues) and file a feature request if needed.

**Reply by bertiebaggio** [reply]
Hi Joel, thanks for answering. I see the feature request from 2020 ([which is based on another QA](https://stackoverflow.com/q/61754243/4647265)) is still open:

https://github.com/vega/vega-lite/issues/6485

It's useful to know it's still not implemented; thanks for the heads-up, much appreciated :) I'll pop comments in the SO QAs for others who come looking.
